includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -532,8 +540,23 @@ public static function output( $post ) {
 	 * Save meta box data.
 	 *
 	 * @param int $order_id Order ID.
+	 * @throws Exception Required request data is missing.
 	 */
 	public static function save( $order_id ) {
+		// phpcs:disable WordPress.Security.NonceVerification.Missing
+
+		if ( ! isset( $_POST['order_status'] ) ) {
+			throw new Exception( __( 'Order status is missing.', 'woocommerce' ), 400 );
+		}
+
+		if ( ! isset( $_POST['_payment_method'] ) ) {
+			throw new Exception( __( 'Payment method is missing.', 'woocommerce' ), 400 );
+		}
+
+		if ( ! isset( $_POST['_wpnonce'] ) ) {
```

Comment:
I think is only missing to remove this one, since we don't check for it in this method.

Reply:
Removed!